### PR TITLE
Add plugin removal docs and a banner for EOL

### LINKED ISSUE #534: End of life
Pulpcore 3.20 is a natural point to cease ongoing maintenance of the migration plugin.

- [ ] Make the announcement
- [x] Ensure the plugin removal process is well documented & tested
- [x] ~Archive the repository (although 6.9 is supported until October so we can't do this immediately)~ File a task to archive the repo

### PR DESCRIPTION
**Pull request by goosemania**
closes #534

### CODE REVIEW

File: docs/_static/eol_banner.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+window.onload = function() {
+    var elem = document.createElement('div');
+    var body = document.getElementsByClassName("bodywrapper")[0]
+    var doc = document.getElementsByClassName("body")[0]
+    elem.className = "admonition important"
+    elem.id = "pulp-2to3-migration-eol-banner"
+    elem.innerHTML = "<p>This plugin reaches its EOL on XXX, the last supported pulpcore" +
```
Comment (goosemania) [inline docs/_static/eol_banner.js:7]: Need to pick a date https://discourse.pulpproject.org/t/pulp-2to3-migration-plugin-end-of-life-eol-date/457

File: CHANGES/534.doc [hunk 2]
```diff
@@ -0,0 +1,2 @@
+Added steps to remove the plugin and announced its EOL.
+Pulp-2to3-migration plugin reaches its EOL on XXX, the last supported pulpcore version is 3.19.
```
Comment (goosemania) [inline CHANGES/534.doc:2]: https://discourse.pulpproject.org/t/pulp-2to3-migration-plugin-end-of-life-eol-date/457

Comment (goosemania) [reply CHANGES/534.doc:2]: @ggainey , could you weigh in here or on the thread please?

File: docs/conf.py [hunk 3]
```diff
@@ -103,7 +103,7 @@
 html_static_path = ['_static']
 
 # Add in a banner
-html_js_files = ['survey_banner.js']
```
Comment (ipanova) [inline docs/conf.py:106]: mind removing survey_banner.js?

**Review by fao89 — APPROVED** [review]

**Pull request merged**